### PR TITLE
Bug 1808605 -  firefox-android: Migrate fenix jobs on git-push and PRs

### PR DESCRIPTION
**Pull request by JohanLorenzo**
⚠️ Do not land before Tuesday, February 14th, 2023.

Depends on #719 
https://bugzilla.mozilla.org/show_bug.cgi?id=1808605

### CODE REVIEW

**Comment by JohanLorenzo** [comment]
@mergifyio backport releases_v110 releases_v111

File: taskcluster/android_taskgraph/transforms/build_apk.py [hunk 1]
```diff
@@ -46,32 +46,59 @@ def add_shippable_secrets(config, tasks):
...
+                            "firebase",
+                            "app/src/{}/res/values/firebase.xml".format(
+                                gradle_build_type
+                            ),
+                        ),
+                        ("sentry_dsn", ".sentry_token"),
+                        ("mls", ".mls_token"),
+                        ("nimbus_url", ".nimbus"),
+                        ("wallpaper_url", ".wallpaper_url"),
+                        ("pocket_consumer_key", ".pocket_consumer_key"),
```
Comment (JohanLorenzo) [inline taskcluster/android_taskgraph/transforms/build_apk.py:73]: Note: These 2 secrets and `firebase` didn't exist in Focus. Staging uses dummy secrets so we may hit an issue that we didn't see during testing.

File: taskcluster/scripts/toolchain/external-gradle-dependencies.sh [hunk 2]
```diff
@@ -41,7 +51,8 @@ if [[ $WORKING_DIR == ${ANDROID_COMPONENTS_DIR}* ]]; then
   done
 fi
 
-./gradlew $GRADLE_ARGS $GRADLE_COMMANDS
+./gradlew $GRADLE_ARGS $GRADLE_REGULAR_COMMANDS
+set +e; ./gradlew $GRADLE_ARGS $GRADLE_TEST_COMMANDS; set -e
```
[the remark this answers is not in the document]
Comment (JohanLorenzo) [reply taskcluster/scripts/toolchain/external-gradle-dependencies.sh:55]: Oh! Note to self: add this comment back ➡️  https://github.com/mozilla-mobile/firefox-android/commit/366d63a61255bec91ed80be70b9a2a777d2b3883#diff-3177697d5268d388e925ae6656d81c7d313182acc5d550277766b166f9daea29L67

File: taskcluster/ci/ui-test-apk/kind.yml [hunk 3]
```diff
@@ -0,0 +1,534 @@
...
+            env:
+                GOOGLE_PROJECT: moz-fenix
+            retry-exit-status: [20]
+        routes:
+            - notify.slack-channel.G016BC5FUHJ.on-failed
+        scopes:
+            - queue:route:notify.slack-channel.G016BC5FUHJ.on-failed
+            - notify:slack-channel:G016BC5FUHJ
+        extra:
+            notify:
```
[the remark this answers is not in the document]
Comment (JohanLorenzo) [reply taskcluster/ci/ui-test-apk/kind.yml:160]: Good call! @AaronMT would you like to have Slack notifications on focus UI tests too?

**Review by gbrownmozilla — APPROVED** [review]

File: taskcluster/ci/ui-test-apk/kind.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (AaronMT) [reply taskcluster/ci/ui-test-apk/kind.yml:160]: Yes please, if not too late. This line `"text": "Fenix :firefox: ${task.metadata.name} :x:\n "` `Fenix` would just have to replaced with the app name (`:firefox:`) is a Firefox emoji on Slack, but I guess that can stay for now.

[the remark this answers is not in the document]
Comment (JohanLorenzo) [reply taskcluster/ci/ui-test-apk/kind.yml:160]: Sounds good! I'll merge this PR as-is and make the change in a follow-up after the migration. Please do not hesitate to ping me if I forgot to do it 🙂

File: taskcluster/scripts/toolchain/external-gradle-dependencies.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (JohanLorenzo) [reply taskcluster/scripts/toolchain/external-gradle-dependencies.sh:55]: Done.

**Pull request merged**